src/vivarium_cluster_tools/psimulate/runner.py

```diff
@@ -107,6 +102,26 @@ def try_run_vipin(log_path: Path) -> None:
         logger.warning(f"Performance reporting failed with: {e}")
 
 
+def run_rq_dashboard(redis_urls: list, output_directory: Path) -> None:
+    # Get generic hostname url
+    url = redis_urls[0].split("//")[1]
+    hostname = url.split(":")[0]
+
+    # Set up log file
+    rq_dashboard_log = (output_directory / "rq.log").open("a")
+    logger.info("Fetching redis urls and starting RQ-Dashboard")
+    split_urls = " -u ".join(url for url in redis_urls)
```

Comment:
Two things:

`" -u ".join(url for url in redis_urls)` is the same as `" -u ".join(redis_urls)`

and 
`url_flags = " ".join([f'-u {url}' for url in redis_urls])`
means you can write
`command = "rq-dashboard " + url_flags + " --debug"`
without the fencepost `-u` in the command.